### PR TITLE
Minor update "Time sync"

### PR DESCRIPTION
**Pull request by DanielHo-BS**
1. Update the images
2. Update the camera trigger

### CODE REVIEW

File: application_note/time_sync/software_configuration.rst [hunk 1]
```diff
@@ -46,49 +46,51 @@ Installation
...
-    sudo python3 isr_camera.py start
-    # Otherwise, trigger cameras by 10 hz and setting GPIO: 6 pin to interrupt.
-    sudo python3 isr_camera.py start 10 6
+.. code-block:: bash
 
-    # Stop the daemon
-    sudo python3 isr_camera.py stop
+    # Sync mode: start the daemon
+    # To trigger camera by 5 Hz with ISR (from GPIO_NUM 5).
+    sudo python3 adlink_camera_sync sync -f 5 -t 5
```
Comment (evshary) [inline application_note/time_sync/software_configuration.rst:77]: Great! Thanks for the update.
Please also update the "Time Synchronization" page.

**Review by evshary — CHANGES_REQUESTED** [review]

File: application_note/time_sync/software_configuration.rst (hunk 1 above)
Comment (DanielHo-BS) [reply application_note/time_sync/software_configuration.rst:77]: OK, I will update this page.

**Comment by DanielHo-BS** [comment]
Update the "Time Synchronization" page.
1. update usage of camera trigger daemon
2. remove the "sensor_subscriber" function from sensors_pkg

**Comment by evshary** [comment]
LGTM.

**Pull request merged**